lib/ddf.rb

```diff
@@ -0,0 +1,21 @@
+module DDF
+  def self.traverse(schema, &block)
+    recure = ->(item) { traverse(item, &block) }
```

Comment:
typo? `recure` => `recurse`?

Reply:
Noticed it was in the original code too :)

Reply:
I thought `recure` is a real word :laughing: from recurring :see_no_evil: gonna change